### ISSUE TITLE
Split documentation in another repository

### ISSUE THREAD

**Issue by Sytten** [graphql-nexus/schema#611]
I have seen quite a few documentations changes for plugins that do live in this repository like the prisma plugin.
As the ecosystem matures, this will only become more and more the case.
So I think the documentation should move to a separate repo that can also better track issues and missing part of it.
Having clear guidelines for the documentation would also be appreciated.

**Comment by jasonkuhrt** [comment]
We'll lose the ability to colocate features and documentation updates. Any thoughts about that? How about documentation versioning?

**Comment by Sytten** [comment]
Yeah thats annoying. Maybe then only the plugin documentation should move somewhere? It just feels weird to have prisma related stuff in nexus main repo (that also includes issues). And then you have a repo to aggregate the doc from plugins and core. That would also allow third party plugins to be integrated in the official doc.

**Comment by jasonkuhrt** [comment]
Yeah there's no great reason to have Prisma docs here. The other plugins have docs here because they ship with the Nexus package.

It would be nice to scrape npm for Nexus plugins and then scrap docs from README.md etc.

If someone is up for writing a draft spec on how that could work that would be welcome.

**Comment by MatthewCaseres** [comment]
> Yeah there's no great reason to have Prisma plugin docs here. The other plugins have docs here because they ship with the Nexus package.
> 
> It would be nice to scrape npm for Nexus plugins and then scrape docs from README.md etc.
> 
> If someone is up for writing a draft spec on how that could work that would be welcome.

I have a framework for building docs that loads markdown files from remote github locations, I should be able to load collections of documentation from various repos without too much extension.

Right now it computes resource locations (md files from raw.githubuser) using relative paths, but I can include absolute paths if there is a need to aggregate plugin docs.

You can easily include links to the GitHub page for each page in your docs too, seen in example below.

Here are the docs: https://www.openedtech.dev/Open-EdTech/next-mdx-books/about.md
